lib/code_corps/stripe_service/stripe_managed_connect_account.ex

```diff
@@ -0,0 +1,17 @@
+defmodule CodeCorps.StripeService.StripeManagedConnectAccountService do
+  alias CodeCorps.StripeService.Adapters.StripeConnectAccountAdapter
+  alias Stripe.Connect.OAuth.TokenResponse
+
+  @api Application.get_env(:code_corps, :stripe)
+
+  def create(%{} = attributes) do
+    with {:ok, stripe_params} <- StripeConnectAccountAdapter.to_stripe_params(attributes),
+         {:ok, %Stripe.Account{} = stripe_account} <- @api.Account.create(stripe_params),
+         {:ok, params} <- StripeConnectAccountAdapter.to_managed_params(stripe_account, attributes)
+    do
+      %CodeCorps.StripeConnectAccount{}
+      |> CodeCorps.StripeConnectAccount.managed_create_changeset(params)
+      |> CodeCorps.Repo.insert
+    end
+  end
+end
```

Comment:
I rewrote some of this with pseudocode in #575. I would consider taking the pattern matching of that function. Would also just use `create_changeset/2` rather than a special one for `managed`, per comment above.